File: 8.js
const isPalindrome = (string) => {
  const lower = string.toLowerCase().replace(/[,.!\s]/g, '');
  return lower === [...lower].reverse().join('');
}
console.log(isPalindrome('fds'));
console.log(isPalindrome('fdsdf'));
console.log(isPalindrome('Wódy żal dla żydów.'));

File: 8.py
def is_palindrome(arg):
  string = str(arg)
  string.strip(' ')
  return reverse(string) == string


def reverse(string): 
    string = string[::-1] 
    return string 

print(is_palindrome('kajak'))
print(is_palindrome('kajak kajak'))
